concent_api/core/validation.py

```diff
@@ -218,3 +226,121 @@ def validate_list_task_to_compute_ids(subtask_results_accepted_list):
     for task_to_compute in subtask_results_accepted_list:
         subtask_ids.append(task_to_compute.subtask_id + ':' + task_to_compute.task_id)
     return len(subtask_ids) == len(set(subtask_ids))
+
+
+def get_validated_client_public_key_from_client_message(golem_message: message.base.Message):
+    if (
+        isinstance(golem_message, message.concents.ForcePayment) and
+        isinstance(golem_message.subtask_results_accepted_list, list) and
+        len(golem_message.subtask_results_accepted_list) > 0
+    ):
+        task_to_compute = get_field_from_message(golem_message.subtask_results_accepted_list[0], 'task_to_compute')
```

Comment:
`get_field_from_message` should not be used for `task_to_compute` field, as there is a helper/property in `GolemMessages` that allows you to retrieve it

Reply:
I didn't write this whole function, I just cut it from utils/helpers and paste it here to fix import errors. If this is some sort of a bug, we should make separate issue for that.

Reply:
Done.